pinc/misc.inc

```diff
@@ -595,6 +595,110 @@ function get_filelist($dirpath, $validext = array(), $with_path = False)
     closedir($d);
     return $filelist;
 }
+
+// -----------------------------------------------------------------------------
+function is_valid_zip_file(string $file_path): bool
+// Return whether the file is a valid zip file.
```

Comment:
Please put these comments above the function declaration instead of between it and the open bracket. I realize that the code is inconsistent in this.